Package.swift

```diff
@@ -46,6 +46,10 @@ let packageDependencies: [Package.Dependency] = [
     url: "https://github.com/apple/swift-nio-extras.git",
     from: "1.4.0"
   ),
+  .package(
+    url: "https://github.com/apple/swift-atomics.git",
+    from: "1.1.0"
```

Comment:
There was a 1.2.0 release last week to support Swift 5.9 - should we depend on that version instead?

Reply:
Good spot. It's actually not needed at all, I started implementing this with atomics but realised you can't clamp atomically so ended up with a lock...